README.md

```diff
@@ -84,11 +84,19 @@ All the paths have to be absolute, as they will be used when the code is run on
 We can then run `./case.setup` and `./case.build`to setup and build the test case, and `./case.submit` to submit the job to the scheduler.
 
 **Note:**  
-By default, CESM will place outputs and logs/restart files on NCAR's supercomputer Derecho in `/glade/scratch/user/archive/case/`.
-Outputs of failed runs can be found in`/glade/scratch/user/case/`.
+By default, CESM will place outputs and logs/restart files on Derecho in `/glade/derecho/scratch/<user>/archive/<case>/`.
+Outputs of failed runs can be found in`/glade/derecho/scratch/<user>/<case>/`.
 
 To place all output with logs in `archive/case` switch 'short term archiving' on by editing `env_run.xml` in the case directory to change `DOUT_S` from `FALSE` to `TRUE` -- you can do this quickly by running `﻿./xmlchange DOUT_S=FALSE`.
 
+### Expected Output
+
+The run will create a `bld` and a `run` directory in the case folder. The `bld` directory contains build files, logs and executables. The `run` directory contains the run logs, namelists and output Netcdf files.
```

Comment:
Not allowing me to make a suggestion for some reason, but "NetCDF".

Reply:
Is there no "role eyes" emoji on here....?

Reply:
done